ci/pipelines/cf-deployment.yml

```diff
@@ -136,6 +154,13 @@ resource_types:
   source:
     repository: cloudfoundry/bosh-deployment-resource
 
+# TODO remove this resource type declaration when a final release of the resource is available
+- name: bosh-io-stemcell
+  source:
+    repository: foundationalinfrastructure/bosh-io-stemcell-resource
+    tag: v1.2.1
+  type: docker-image
```

Comment:
What is the difference between this resource type and the one we were already using? Is it okay that all of our `bosh-io-stemcell` resources start using this type instead of the old one?

Reply:
The new version of "bosh-io-stemcell-resource" supports authenticated download. There is also an official release v1.3.0: https://github.com/concourse/bosh-io-stemcell-resource/releases/tag/v1.3.0. It's hosted in the DockerHub "concourse" repository: https://hub.docker.com/layers/concourse/bosh-io-stemcell-resource/1.3.0/images/sha256-4badbf26fa9ef824d5963ac2a2ce76c0b5e3d2c37a455373bbaa20cc759445d8?context=explore. I'll try to switch to that one so we are not relying on the "foundationalinfrastructure" repo.

There is unfortunately no Concourse release available which comes with the v1.3.0 as bundled resource. The latest Concourse release v7.11.2 still uses "bosh-io-stemcell-resource" v1.2.0: https://github.com/concourse/concourse/releases/tag/v7.11.2